test/com/amazon/ion/impl/lite/BaseIonSequenceLiteSublistTestCase.java

```diff
@@ -44,6 +44,11 @@
 
     protected abstract IonSequence newSequence();
 
+    List<IonValue> newSubList() {
+        List<IonValue> seq = newSequence();
+        return seq.subList(0, seq.size());
+    }
```

Comment:
This is a bizarre method, isn't it more `newEmptySequenceSubList` or some such?

Reply:
Well, `newSequence()` always returns a `SEXP`, `LIST`, or `DATAGRAM` (depending on the deriving test class (`IonListSexpLiteSublistTest` or `IonDatagramLiteSublistTest`) with elemnets for integer values 0-7. 

So, no, it's not am empty sublist here... just one that covers the entire range of the sequence.

Reply:
Ah, my misunderstanding.